wikipedia/README.md

```diff
@@ -0,0 +1,58 @@
+## Wikipedia Search track
+
+This track benchmarks
+
+The dataset is derived from a dump of wikipedia availaible here:
+https://dumps.wikimedia.org/enwiki/latest/enwiki-latest-pages-articles-multistream.xml.bz2.
+
+Each page is formatted into a JSON document with the following fields:
+
+title: Page title
+namespace: Optional namespace for the page. [Namespaces](https://en.wikipedia.org/wiki/Wikipedia:Namespace) allow for the organization and separation of content pages from administration pages.
+content: Page content.
+redirect: If the page is a redirect, the target of the redirection. In this case content is empty.
+
+Fields that do not have values have been left out.
+
+### Generating the documents dataset
+
+To regenerate the dataset from scratch, first download and unzip an archive
+of Wikipedia dumps from [this link](https://dumps.wikimedia.org/enwiki/latest/enwiki-latest-pages-articles-multistream.xml.bz2) (~21GB).
+
+Then run this command:
+
+```bash
+python _tools/parse_documents.py <path_to_xml_file> | bzip2 --best > documents.json.bz2
```

Comment:
```suggestion
python _tools/parse_documents.py <path_to_xml_file> | pbzip2 -9 -k -m2000 > documents.json.bz2
```

Reply:
see https://github.com/elastic/rally-tracks/pull/429/files#r1294578331 and https://esrally.readthedocs.io/en/stable/adding_tracks.html?highlight=bzip2#sharing-your-track-with-others for more details